internal/provisioner/kubecost.go

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package provisioner
+
+import (
+	"github.com/mattermost/mattermost-cloud/k8s"
+	"os"
+	"strings"
+
+	"github.com/mattermost/mattermost-cloud/internal/tools/aws"
+	"github.com/mattermost/mattermost-cloud/internal/tools/kops"
+	"github.com/mattermost/mattermost-cloud/model"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+type kubecost struct {
+	awsClient      aws.AWS
+	environment    string
```

Comment:
Some fields in this struct seem not to be used, such as `awsClient`, `environment`, and `cluster`.  If those are not needed I would suggest removing them.